editor/template_builders.py

```diff
@@ -16,7 +16,7 @@ def parse_template(inherits, source, delimiter):
     meta_prefix = delimiter + " meta-"
     meta = ["name", "description", "version", "space-indent"]
 
-    with open(source) as f:
+    with open(source, "r", encoding="utf-8") as f:
```

Comment:
For consistency, but these are `GDScript` files, not sure what the encoding should be

Reply:
Everything should be UTF-8 in the repo.